data/paper.org

```diff
@@ -74,9 +74,9 @@ C(v_0, v_f) = S(gain(v_0, v_f),\ max(v_0),\ stp(v_0))
 Where:
 \begin{align*}
 gain(v_0, v_f) =& \ v_f - v_0\\
-max(v_0) =& \ 5 - \frac{4}{5} \log_{10}(v_0 + 1)\\
-mid(v_0) =& \frac{77}{1000}v_0 + 75 \\
-stp(v_0) =& \frac{0.04}{\left( \frac{v_0}{200} + 1 \right)}  
+max(v_0) =& \ 1.2 + \frac{1.9}{\frac{v_0}{10}+1}\\
+mid(v_0) =& \frac{2}{125}v_0 + 100 \\
+stp(v_0) =& \frac{0.04}{\frac{v_0}{100} + 1}  
 \end{align*}
 
 The maximum value, $max(v_0)$, represents an investment's maximum possible return. This maximum 
```

Comment:
Should be updated to say the maximum decreases as the inverse of $v_0$

Reply:
ahh I missed that!